azure-pipelines.yml

```diff
@@ -13,6 +13,11 @@ parameters:
   - nimodinst
   - nise
   - niswitch
+- name: rhel_module_names
+  type: object
+  default: # Order of the modules is based on test execution time, longest first.
+  - nidmm
+  - niswitch
```

Comment:
More than these two drivers are supported in RHEL.
Please create an issue to add the rest of the supported drivers:

1. NI-DCPower
1. NI-SCOPE
1. NI-TClk

Reply:
https://github.com/ni/nimi-python/issues/1669